src/MarginTrading.Backend.Services/NotificationSenderBase.cs

```diff
@@ -60,17 +63,18 @@ private string GetPushMessage(IOrder order)
             var message = string.Empty;
             var volume = Math.Abs(order.Volume);
             var type = order.GetOrderType() == OrderDirection.Buy ? "Long" : "Short";
-
+            var instrumentName = _assetPairsCache.TryGetAssetPairById(order.Instrument)?.Name ?? order.Instrument;
+            
             switch (order.Status)
             {
                 case OrderStatus.WaitingForExecution:
-                    message = string.Format(MtMessages.Notifications_PendingOrderPlaced, type, order.Instrument, volume, Math.Round(order.ExpectedOpenPrice ?? 0, order.AssetAccuracy));
+                    message = string.Format(MtMessages.Notifications_PendingOrderPlaced, type, instrumentName, volume, Math.Round(order.ExpectedOpenPrice ?? 0, order.AssetAccuracy));
                     break;
                 case OrderStatus.Active:
                     message = order.ExpectedOpenPrice.HasValue
-                        ? string.Format(MtMessages.Notifications_PendingOrderTriggered, order.GetOrderType() == OrderDirection.Buy ? "Long" : "Short", order.Instrument, volume,
+                        ? string.Format(MtMessages.Notifications_PendingOrderTriggered, order.GetOrderType() == OrderDirection.Buy ? "Long" : "Short", instrumentName, volume,
```

Comment:
order.GetOrderType() == OrderDirection.Buy ? "Long" : "Short"
1. Expression already evaluated as type. Better to reuse it.
2. Better to move consts to a separate static resource.